instance/tests/models/test_instance.py

```diff
@@ -705,6 +705,7 @@ def test_provision_failed_exception_email(self):
         self.assertEqual(mime_type, "text/html")
 
 
+# pylint: disable=too-many-public-methods
```

Comment:
@e-kolpakov Thanks! And not sure why it comes up in your PRs : ) I don't think the pylint suppressions I saw in the upgrade were from you :p although I haven't checked.

@smarnach You can always still split in smaller chunks, tests can be group logically. If anything it will make it easier to find the right test, just by categorizing the tests better.

For pylint in general, I know sometimes it can look frivolous or counter productive to address the warnings, but avoiding pylint exclusions has the merit of making it easier to review them, and to have a more unified code style. And they aren't usually that hard to follow.